generators/app/templates/src/services/utils.ts

```diff
@@ -0,0 +1,131 @@
+import { useState, useEffect, useCallback } from 'react';
```

Comment:
This looks great!

Maybe we can name the file requests or something like that instead of utils?

Reply:
I'll take the suggestion from @LucasZibell and move them to a hooks folder to encourage creating custom hooks. I think I'll separate them too.